gulp/tasks/styles.js

```diff
@@ -16,8 +17,10 @@ gulp.task('styles', function () {
       sourceMap: 'sass',
       outputStyle: global.isProd ? 'compressed' : 'nested'
     }))
+    .pipe(gulpif(config.logsizes, size({showFiles: true, title:'Pre-autoprefixer'})))
     .pipe(autoprefixer("last 2 versions", "> 1%", "ie 8"))
     .on('error', handleErrors)
+    .pipe(gulpif(config.logsizes,size({showFiles: true, title:'Post-autoprefixer'})))
```

Comment:
could you add a space between the `gulpif` arguments, and again put the options on separate lines?